skyportal/models.py

```diff
@@ -86,6 +87,8 @@
 _, cfg = load_env()
 cosmo = establish_cosmology(cfg)
 
+ws_flow = Flow()
```

Comment:
@stefanv Can there be more than one `Flow` instance in the app like this, or do I need to import the instance created in `baselayer`?